activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -307,7 +307,7 @@ module ClassMethods
       # [:allow_destroy]
       #   If true, destroys any members from the attributes hash with a
       #   <tt>_destroy</tt> key and a value that evaluates to +true+
-      #   (e.g. 1, '1', true, or 'true'). This option is off by default.
+      #   (e.g. 1, '1', true, or 'true'). This option is +false+ by default.
```

Comment:
Nitpick: I think "false" here should not use monospace because it is [not referring to the `false` singleton](https://guides.rubyonrails.org/v7.1/api_documentation_guidelines.html#booleans).  In this case, the default is actually (probably) `nil`, but that does not matter to the user, so we can say "false" to contrast with "If true".

Reply:
> In this case, the default is actually (probably) `nil`

@jonathanhefner I might be misreading the implementation, but the default options declare `allow_destroy: false`:

https://github.com/rails/rails/blob/8d778e0708196bdac2ce0d18e24600735469f1a3/activerecord/lib/active_record/nested_attributes.rb#L351-L355

I'm happy to remove the wrapping `+` characters, but I want to make sure that I understand the implications first.

Reply:
`+false+` is inline with the description of `:update_only`: `By default the +:update_only+ option is +false+`.
Maybe also change `If true, destroys ` to `If +true+, destroys `, although I don't have a strong opinion here.

Reply:
Reading the link from @jonathanhefner I agree with his suggestion.

Reply:
@jonathanhefner thank you for sharing the link to https://guides.rubyonrails.org/v7.1/api_documentation_guidelines.html#booleans, I had no idea there were style guides for documentation! I'll read up on the rest of what's provided there.

In the meantime, with the Boolean documentation in mind, is this diff valuable at all? Is "false" an improvement over "off"? Should I replace `+false+` with `false`, or would it be better to close this without change?

Reply:
We should not use `+false+`.  I don't have a strong opinion on "false" vs "off", but if you think it reads more clearly, then I'm :+1:.